index.bs

```diff
@@ -571,6 +581,8 @@ The <dfn attribute for="XRRenderState">depthNear</dfn> attribute defines the dis
 
 {{XRRenderState/depthNear}} and {{XRRenderState/depthFar}} is used in the computation of the {{XRView/projectionMatrix}} of {{XRView}}s and determines how the values of an {{XRWebGLLayer}} depth buffer are interpreted. {{XRRenderState/depthNear}} MAY be greater than {{XRRenderState/depthFar}}.
 
+The <dfn attribute for="XRRenderState">inlineVerticalFieldOfView</dfn> attribute defines the default vertical field of view in radians used when computing projection matrices for {{XRSessionMode/inline}} {{XRSession}}s. The projection matrix calculation also takes into account the aspect ratio of the {{XRRenderState/outputContext}}'s {{XRPresentationContext/canvas}}. This value MUST be <code>null</code> for {{XRSessionMode/immersive-vr}} and {{XRSessionMode/immersive-ar}} {{XRSession}}s.
```

Comment:
>This value MUST be <code>null</code> for {{XRSessionMode/immersive-vr}} and {{XRSessionMode/immersive-ar}} {{XRSession}}s.

Let's just use the immersive session property instead of calling out both types.

Reply:
Done